.gitignore

```diff
@@ -24,6 +24,15 @@ coverage.cov
 # Likewise, we don't want to keep the typechecked version of our codegen tests.
 pkg/codegen/internal/test/testdata/*/typecheck/**
 
+# We don't need to keep testing python __pycache__ either either
+pkg/codegen/internal/**/__pycache__
+pkg/codegen/internal/test/testdata/*-pp/*/*.csproj
+pkg/codegen/internal/test/testdata/*-pp/*/go.mod
+pkg/codegen/internal/test/testdata/*-pp/*/go.sum
+pkg/codegen/internal/test/testdata/*-pp/*/Program.cs
+pkg/codegen/internal/test/testdata/*-pp/*/package.json
+pkg/codegen/internal/test/testdata/*-pp/*/go.sum
```

Comment:
I thought that the `__pycache__` line at the bottom would catch `__pycache__` at any level of nesting. Does it not?

Reply:
I think it does.